packages/devtools-launchpad/package.json

```diff
@@ -14,6 +14,7 @@
   "scripts": {
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
+    "license-check": "../devtools-license-check/bin/devtools-license-check",
```

Comment:
I guess not since you also say it's symlinked after that.  I don't understand why it would be this way.